multigeiger/speaker.cpp

```diff
@@ -106,10 +108,13 @@ void IRAM_ATTR isr_audio() {
     // duration == 0 marks the end of the sequence to play
     portENTER_CRITICAL_ISR(&mux_audio);
     isr_sequence = NULL;
-    if (playing_tick)
+    if (playing_tick) {
       isr_tick_sequence = NULL;
-    else if (playing_audio)
+      playing_tick = false;
```

Comment:
... and the second bugfix is that `playing_tick `and `playing_audio` were never reset to false, being declared as static. Thus the reset here. Will also be part of the new bugfix-PR, after IotWebConf-Fix, @ThomasWaldmann.

Reply:
oh, oops, yeah - confirmed. X-)

Reply:
I changed the static variables to normal disposable ones. That feels more straightforward.

Reply:
see the other PR.